static/_redirects

```diff
@@ -28,7 +28,7 @@
 /docs/admin/audit/     /docs/tasks/debug-application-cluster/audit/ 301
 /docs/admin/authorization/rbac.md     /docs/admin/authorization/rbac/ 301
 /docs/admin/cluster-components/     /docs/concepts/overview/components/ 301
-/docs/admin/cluster-management/     /docs/tasks/administer-cluster/cluster-management/ 301
+/docs/admin/cluster-management/     /docs/tasks/administer-cluster/ 307
 /docs/admin/cluster-troubleshooting/     /docs/tasks/debug-application-cluster/debug-cluster/ 301
```

Comment:
nit: Check whether netlify will redirect this page (307)?

Reply:
That's annoying. 307 has been in HTTP since 1999, I just assume it's available everywhere. I will revise.